### PR TITLE
[FIX] Selected messages don't get unselected

### LINKED ISSUE #20407: Selected messages don't unselect on clicking
### Description:

Selected messages don't unselect while exporting via email

### Steps to reproduce:

See video

### Expected behavior:

The blue selection must go away.
### Actual behavior:
https://user-images.githubusercontent.com/64399555/105844319-0fb8ca80-5fff-11eb-92e0-15510ede3c7e.mp4

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by im-adithya**
## Proposed changes (including videos or screenshots)

https://user-images.githubusercontent.com/64399555/105844776-c157fb80-5fff-11eb-90cc-94e9f69649b6.mp4

## Issue(s)

This PR closes #20407

### CODE REVIEW

File: client/views/room/contextualBar/ExportMessages/index.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ const MailExportForm = ({ onCancel, rid }) => {
 
 	const reset = useMutableCallback(() => {
 		setSelected([]);
-		$(`#chat-window-${ rid }.messages-box .message.selected`)
+		$('.messages-box .message', $(`#chat-window-${ rid }`))
```
Comment (gabriellsh) [inline client/views/room/contextualBar/ExportMessages/index.js:132]: A simple space between them would have sufficed, but this achieves the same result. Thanks!

**Review by gabriellsh — APPROVED** [review]

File: client/views/room/contextualBar/ExportMessages/index.js (hunk 1 above)
Comment (im-adithya) [reply client/views/room/contextualBar/ExportMessages/index.js:132]: I tried the same first but it didn't work, perhaps I did some errors.
Btw always welcome! 😁

**Comment by im-adithya** [comment]
Hey @gabriellsh! Can you please merge this and review my other PRs so that I can make necessary changes and get them merged as well...?

**Comment by gabriellsh** [comment]
@im-adithya Please, be patient. It'll be merged when it's supposed to. We review PRs as soon as we have enough time for it, no need to ask or tag anyone.

Thanks.

**Pull request merged**